020-Connect/015-CLI/010-cli.mdx

```diff
@@ -11,6 +11,35 @@ Xata provides a CLI (command line interface) through an **[NPM package](https://
 
 ## Installation
 
+### Install with an installer
+
+### Macos
+
+- Install Homebrew then run:
+
+`brew tap xata/brew && brew install xata` (TODO)
+
+### Windows
+
+- [64-bit installer](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-x64.exe)
+- [32-bit installer](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-x86.exe)
+
+### Standalone Tarball containing Binary
+
+- We provide the binaries in .gz format. The tarball contains Node.js.
+
+- Setup script for tarball users (TODO)
+
+- [macOS (m1)](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-darwin-arm64.tar.gz)
+- [macOS (Intel)](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-darwin-x64.tar.gz)
+- [Linux (x64)](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-linux-x64.tar.gz)
+- [Linux (arm)](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-linux-arm.tar.gz)
+- [Linux (arm64)](https://xata-cli-assets.s3.us-east-1.amazonaws.com/channels/stable/xata-linux-arm64.tar.gz)
```

Comment:
Not sure if this is the correct info in parenthesis, is arm vs arm64 enough/meaningful?